src/lib.rs

```diff
@@ -4,7 +4,7 @@
 //! Section references in further documentation refer to this document.
 //! https://www.intel.com/content/dam/www/public/emea/xe/en/documents/manuals/64-ia-32-architectures-software-developer-vol-3d-part-4-manual.pdf
 
-#![no_std]
+#![cfg_attr(not(test), no_std)]
```

Comment:
This could have been a fixup for the commit, where you removed the `std` feature.